app/repositories/packages_repository.rb

```diff
@@ -3,6 +3,9 @@
 class PackagesRepository < RmapiRepository
   def find!(id)
     vendor_id, package_id = id.split('-')
+
+    fail RequestError.new('Package and vendor id are required', 400) unless vendor_id && package_id
```

Comment:
Can we also change this error message to `Package and provider id are required` because consumer of mod-kb-ebsco would not know what a vendor is...its an RM API term serialized to `provider` in mod-kb-ebsco